app/presenters/service_actions_presenter.rb

```diff
@@ -20,8 +20,12 @@ def actions(product)
       actions << { name: 'Applications', path: admin_service_application_plans_path(product) }
     end
 
-    actions << { name: 'ActiveDocs', path: admin_service_api_docs_path(product) } if can?(:manage, :plans)
-    actions << { name: 'Integration', path: admin_service_integration_path(product) } if can?(:manage, :plans)
+    if can?(:manage, :plans)
+      actions << { name: 'ActiveDocs', path: admin_service_api_docs_path(product) }
+      actions << { name: 'Integration', path: admin_service_integration_path(product) }
+    elsif can?(:manage, :policy_registry)
+      actions << { name: 'Policies', path: edit_admin_service_policies_path(product) }
+    end
```

Comment:
Shouldn't this be like:
```suggestion
    if can?(:manage, :plans)
      actions << { name: 'ActiveDocs', path: admin_service_api_docs_path(product) }
      actions << { name: 'Integration', path: admin_service_integration_path(product) }
    end
    
    if can?(:manage, :policy_registry)
      actions << { name: 'Policies', path: edit_admin_service_policies_path(product) }
    end
```

Can't user manage both?

Reply:
Yes, they can, and at first I wanted to do it as you are suggesting. But, I think that having both "Integration" and "Policies" items is redundant, because "Policies" is a sub-section of "Integration". So, if we have access to the parent section ("Integration"), we have just that one. And if we don't have access to "Integration", but only to its child "Policies", then we have "Policies" in the list.